integrity-backend/input-chatbot-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```diff
@@ -0,0 +1,44 @@
+{
+  "contentMetadata":
+  {
+    "name": "Telegram archive on 2022-03-29T13:00:00Z",
+    "mime": "application/zip",
+    "description": "Archive of [ 'channel1', 'channel2' ] by Telegram bot starting on 2022-03-29T13:00:00Z",
+    "author":
+    {
+      "@type": "Organization",
+      "identifier": "https://starlinglab.org",
+      "name": "Starling Lab"
+    },
+    "dateCreated": "2022-03-29T13:54:20Z",
+    "extras":
+    {
+      "channels": [
+        "channel1",
+        "channel2"
+      ],
+      "dateRage":
+      {
+        "from": "2022-03-29T13:00:00Z",
+        "to": "2022-03-29T14:00:00Z"
+      }
+    },
+    "private":
+    {
+      "signal":
+      {
+        "phoneNumber": "000-000-0000"
+      },
+      "slack":
+      {
+        "botAccount": "slackArchiveBot",
+        "workspace": "slackWorkSpace"
+      },
+      "telegram":
+      {
+        "botAccount": "telegramArchiveBot"
+      }
```

Comment:
Still need a place to actually store this in. The bot code itself will not provide this information